### PR TITLE
Add ability for clients to send error log to server

### PR DESCRIPTION
**Pull request by nvkevlu**
Renames the error log file to error_log.txt and adds a way to transmit the error logs from clients to the server.

### Description

Renames the error log file to error_log.txt and adds a way to transmit the error logs from clients to the server. Currently should_report_error_log is set to False. This can be set to True to test the feature for now, and in the future we will want to add a way to configure this for each site.

On the server, the error log is simply printed, but future PRs will modify this to save the error log in the proper place.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/private/fed/server/fed_server.py [hunk 1]
```diff
@@ -665,6 +671,11 @@ def quit_client(self, request: Message) -> Message:
             headers = {CellMessageHeaderKeys.MESSAGE: "Removed client"}
             return self._generate_reply(headers=headers, payload=None, fl_ctx=fl_ctx)
 
+    def process_error_log(self, request: Message):
+        payload = request.payload
+        client = request.get_header(key=MessageHeaderKey.ORIGIN)
+        print(f"Error log from {client}: {payload}")
```
Comment (YuanTingHsieh) [inline nvflare/private/fed/server/fed_server.py:677]: Is this for debugging purpose?

File: nvflare/private/fed/client/client_executor.py [hunk 2]
```diff
@@ -393,6 +395,31 @@ def _wait_child_process_finish(self, client, job_id, allocated_resource, token,
             return_code = get_return_code(job_handle, job_id, workspace, self.logger)
 
             self.logger.info(f"run ({job_id}): child worker process finished with RC {return_code}")
+
+            should_report_error_log = (
+                False  # set this to True to report error log to server, todo: get value from client config
```
Comment (YuanTingHsieh) [inline nvflare/private/fed/client/client_executor.py:400]: Do we plan on implement this in this PR?

File: nvflare/private/fed/server/fed_server.py (hunk 1 above)
Comment (nvkevlu) [reply nvflare/private/fed/server/fed_server.py:677]: Yes, this is going to be changed in the next PR for handling this on the server.

File: nvflare/private/fed/client/client_executor.py (hunk 2 above)
Comment (nvkevlu) [reply nvflare/private/fed/client/client_executor.py:400]: Not this one, this will be implemented in a future PR

**Review by SYangster — APPROVED** [review]
LGTM

File: nvflare/fuel/f3/cellnet/defs.py [hunk 3]
```diff
@@ -170,6 +170,7 @@ class CellChannelTopic:
     EXECUTE_RESULT = "execute_result"
     FIRE_EVENT = "fire_event"
     REPORT_JOB_FAILURE = "report_job_failure"
+    TRANSMIT_ERROR_LOG = "transmit_error_log"
```
[the remark this answers is not in the document]
Comment (nvkevlu) [reply nvflare/fuel/f3/cellnet/defs.py]: Fixed

**Review by yanchengnv — APPROVED** [review]
LGTM

**Pull request merged**